scrapy/pipelines/images.py

```diff
@@ -128,19 +138,17 @@ def get_images(self, response, request, info):
             yield thumb_path, thumb_image, thumb_buf
 
     def convert_image(self, image, size=None):
-        if image.format == 'PNG' and image.mode == 'RGBA':
```

Comment:
Yes, the behaviour is same, it makes the transparent images have a black background.